Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM mcr.microsoft.com/vscode/devcontainers/rust:0-1
 
 RUN rustup target add wasm32-unknown-unknown
```

Comment:
I've just noticed that rust is not used to build wasm dependencies.
Perhaps we could use a more basic image.

Reply:
The wasm/Rust-related setup can be removed (thanks to #194 and #203). [The basic deno image](https://github.com/denoland/deno_docker) is a better base.